docs-src/cloud/api-keys/manage-api-keys.md

```diff
@@ -0,0 +1,122 @@
+---
+id: manage-api-keys
+title: Manage API Keys
+sidebar_label: Manage API Keys
+description: Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
+tags:
+  - how to
+---
+
+Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
+
+### Generate an API Key
+
+:::note
+
+Once generated, copy and securely save the API Key.
+It will be displayed only once for security purposes.
+
+:::
+
+Generate an API Key using one of the following methods.
+
+#### Using the Cloud UI
+
+To generate an API Key using the Cloud UI:
+
+1. [Login](https://cloud.temporal.io/) to the Cloud UI.
+2. Select **Settings** and choose **API Keys**.
+3. Select **Create API Key** and provide the following:
+   1. Name
+   2. Description
+   3. Expiration Date
+4. Select **Generate API Key**.
+
+:::note
+
+Temporal supports up to a 90-day duration for API Keys.
+
+:::
+
+#### Using the tcld
+
+To generate an API Key using tcld, use the `tcld apikey create` command.
+
+1. Authenticate your Temporal Cloud account using the [login](/cloud/tcld/login) command.
+2. Create an API Key using the [apikey create](/cloud/tcld/apikey/create) command.
+
+For example:
+
+```command
+tcld login
+tcld apikey create --name Your-Key-Name --description YourDescription --duration 24h
+```
+
+:::note
+
+Use tcld to create API Keys with a day level expiration by day.
+
+:::
+
+### Delete an API Key
+
+Delete an API Key using one of the following methods.
+
+#### Using the Cloud UI
+
+To delete an API Key using the Cloud UI
+
+1. [Login](https://cloud.temporal.io/) to the Cloud UI.
+2. Select **Settings** and choose **API Keys**.
+3. Select the three vertical dots to the right of the API Key’s row and choose **Delete**.
+4. Choose **Delete**.
+
+#### Using the tcld
+
+To delete an API Key using tcld, pass the API Key ID to the [tcld apikey delete](/cloud/tcld/apikey/delete) command.
+
+1. Authenticate your Temporal Cloud account using the [login](/cloud/tcld/login) command.
+2. Delete the API Key using the [apikey delete](/cloud/tcld/apikey/delete) command.
+
+```command
+tcld apikey delete --id <your-api-key-id>
+```
+
+### Update access to an API Key
+
+You can update access to an API Key by enabling or disabling it.
+
+After disabling an API Key, that API Key will no longer be able to authenticate with Temporal Cloud.
+Only enabled keys can authenticate with Temporal Cloud.
+
+#### Using the Cloud UI
+
+To update access to an API Key using the Cloud UI, follow these steps:
+
+1. [Login](https://cloud.temporal.io/) to the Cloud UI.
```

Comment:
Same feedback as above for Adding an API Key. Please use the user guidance [here](https://www.notion.so/temporalio/Docs-for-API-Keys-78085708c5d64edd894dbb69ea9b82a5?pvs=4#69eaf29a2838497e8c8b9b1b128750a4) as only Global Admins can access Settings.

To disable/enable an API Key using the Cloud UI

1. [[Login](https://cloud.temporal.io/)](https://cloud.temporal.io/) to the Cloud UI
2. Navigate to [[Profile Page → API Keys](https://cloud.temporal.io/profile/api-keys)](https://cloud.temporal.io/profile/api-keys)
3. Expand the vertical ellipsis to the right of the API Key’s row
4. Click Disable or Enable